models/comment.js

```diff
@@ -1,13 +1,12 @@
 const mongoose = require('mongoose');
 
-// Deck schema
 const commentSchema = mongoose.Schema({
-  parent: String,
+  parent: mongoose.Schema.Types.ObjectId,
```

Comment:
We store Scyrfall Card-IDs here (or maybe oracle Ids I didn't check closely) so we can't make this an ObjectId field. See CardPage and as you drill down to CommentsSection you'll see how parent gets defined.